vowpalwabbit/text_parser/src/parse_example_text.cc

```diff
@@ -54,6 +54,7 @@ int read_features_string(VW::workspace* all, io_buf& buf, VW::multi_ex& examples
   return static_cast<int>(num_bytes_consumed);
 }
 
+// TODO what to do about this is this details? probably yes
 template <bool audit>
```

Comment:
This class is only used in the cc file, so it should be wrapped in an anonymous namespace so its TU local